### PR TITLE
Add argument to specify which module gets feature report

### PR DESCRIPTION
**Pull request by Kody420**

### CODE REVIEW

File: modpad_service/src/main.rs [hunk 1]
```diff
@@ -30,10 +30,11 @@ fn main() {
     let mut prev_sliders_data: Vec<u8> = vec![0u8;ModpadApi::SLIDER_COUNT.into()];
     loop {
         let sliders_data = modpad_api.read_sliders().expect("Failed ot read sliders");
+        log::info!("{:?}",sliders_data);
```
Comment (M4rc311o) [inline modpad_service/src/main.rs:33]: Is this *logging* still needed to be part of the code? If so, is there any way to do it without spamming the log so much? If it is supposed to be there in this form at least make it a `log::debug!`.

File: src/lib.rs [hunk 2]
```diff
@@ -153,10 +177,19 @@ pub enum Brightness {
     Decrease
 }
 
+#[derive(Clone, ValueEnum, Debug)]
+pub enum Module {
+    Modpad,
+    Module1,
+    Module2,
+    Module3
+}
```
Comment (M4rc311o) [inline src/lib.rs:186]: I am not sure if making *enum* is neccessary when modules are named *ModuleX* (*X* being just a number), and not something dynamically assigned to where correct module is connected. And even if this is the case `#[repr(u8)]` attribute can be used to assign numbers to individual *enum* variants and than all `match module` blocks can be substituted with `module as u8` expression.

**Review by M4rc311o — CHANGES_REQUESTED** [review]

File: modpad_service/src/main.rs (hunk 1 above)
Comment (Kody420) [reply modpad_service/src/main.rs]: Removed it but could be nice to activate it with config or something like that.

Comment (M4rc311o) [reply modpad_service/src/main.rs]: Ok, *env_logger* crate used in this  project uses `RUST_LOG` environment varible to set a filter level. So logger can be initialized without the filter level (`env_logger::init();`) and than just use `RUST_LOG` varible to set the filter level. Feel free to create PR with this change implemented. Check [this](https://docs.rs/env_logger/latest/env_logger/index.html) for reference.

File: src/lib.rs [hunk 3]
```diff
@@ -153,10 +157,20 @@ pub enum Brightness {
...
 }
 
+#[derive(Clone, ValueEnum, Debug)]
+#[repr(u8)]
+pub enum Module {
+    Modpad,
+    Down,
+    Left,
+    Right
+}
```
Comment (M4rc311o) [inline src/lib.rs:167]: As underlying code is depending on values assigned to individual *enum* kinds and also to improve code readability. I would suggest to explicitly assign value to each kind. Suggestion below should be correct, but it is not tested.

```suggestion
#[derive(Clone, ValueEnum, Debug)]
#[repr(u8)]
pub enum Module {
    Modpad = 0x00,
    Down = 0x01,
    Left = 0x02,
    Right = 0x03,
}
```

**Pull request merged**